README.md

```diff
@@ -110,42 +130,58 @@ Follow the prompts and select the starter kit of interest (`Node.js Express App`
 - Look for the link in the App details view for your app in the IBM Cloud console.
-- Look in the deployment logs that are produced by the deployment stage of your automated pipeline. See [Debugging your cloud app](#Debugging-your-cloud-app) for details about how to find these logs.
+- Look in the deployment logs that are produced by the deployment stage of your automated pipeline. See [Debugging your cloud app](#Debugging-build-and-deploy-problems) for details about how to find these logs.
 - Look at the terminal output after you successfully run the `ibmcloud dev create` command.
 
 Follow the app link or paste the URL into a browser to see the landing page for your new microservice. Check the endpoints that are provided by the microservice.
 
-## Debugging your cloud app
-
-### Pipeline logs
+### Debugging build and deploy problems
 
 If your app is not working, check your pipeline status. Problems with build and deploy are displayed there, and from there you can view build and deploy logs. The pipeline is found in the DevOps toolchain for your app. You can find the toolchain in these ways:
 - Look for the link in the App details view for your app in the IBM Cloud console.
 - From the IBM Cloud main dashboard, select DevOps from the navigation menu, then select the region in which your toolchain resides. Find your toolchain in the list, and click it reveal the toolchain components. Click the pipeline component. You see links to logs in the tile for each stage in the pipeline.
 
-### Container logs
+### Debugging your app in cloud container
 
 Container logs hold the information that's coming from your running app, similar to what you would see in the terminal when you run your app locally. Use the kubectl CLI to fetch logs for the problematic pod. For more information, see the [docs for the `kubectl logs` command](https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands#logs).
 
 ## Developing locally
 
+### Cloning the source code
+
+When you set up your IBM Cloud development environment, and source code repo was created for you.  Clone the source code to your local machine. You can find the link to the source code repo in the App Details view for your app in the IBM Cloud console.  
+
+### Working with local credentials
+
+This app does not connect to any cloud services as is, but you can add services yourself.
+
+If you added services to this app using the IBM Cloud developer workflow, download a local credentials file.
+
+```bash
+ibmcloud dev get-credentials
+```
+
+This will download the file `localdev-config.json` which goes in the `server` directory of your app.  If you are using IBM Cloud CLE developer tools, you don't need to add credentials to your local environment.  The CLI will do that for you.
```

Comment:
`The CLI will do that for you.` 
Maybe instead: `The CLI tools will do that for you.`

Reply:
done